src/Lifi.ts

```diff
@@ -213,15 +265,29 @@ class LIFI {
     }
   }
 
+  /**
+   * Get a list of active routes.
+   * @return {Route[]} A list of routes.
+   */
   getActiveRoutes = (): Route[] => {
     return Object.values(this.activeRoutes).map((dict) => dict.route)
   }
 
+  /**
+   * Get a single updated route. The route has to be active.
+   * @param {Route} route - A route object.
+   * @return {ReturnValueDataTypeHere} Brief description of the returning value here.
+   */
   getActiveRoute = (route: Route): Route | undefined => {
     return this.activeRoutes[route.id].route
   }
 
-  // Balances
+  /**
+   * Returns the balances of a specific token a wallet holds across all chains we have aggergated. .
+   * @param {string} walletAddress - A walletaddress.
```

Comment:
Do we have a naming convention here already? I think "wallet address" is a bit more common?

Reply:
in other places here we used "wallet address"